### PR TITLE
Add debug output if before() and setup() called

### PR DESCRIPTION
**Pull request by tekka007**
- Adding debug output if before() and/or setup() are called provides useful information for troubleshooting

### CODE REVIEW

**Comment by tekka007** [comment]
Jenkins, build examples please

File: core/MyGatewayTransportSerial.cpp [hunk 1]
```diff
@@ -42,7 +42,7 @@ bool gatewayTransportSend(MyMessage &message) {
 bool gatewayTransportInit() {
 	gatewayTransportSend(buildGw(_msgTmp, I_GATEWAY_READY).set("Gateway startup complete."));
 	// Send presentation of locally attached sensors (and node if applicable)
-	presentNode();
+	// presentNode();
```
Comment (MartinHjelmare) [inline core/MyGatewayTransportSerial.cpp]: Why comment this?

Comment (tekka007) [inline core/MyGatewayTransportSerial.cpp]: Ups, shouldn't be commented - this will be part of a different PR :) 
Thanks for reporting.
